apps/src/code-studio/pd/application_dashboard/admin_cohort_view.jsx

```diff
@@ -63,6 +64,18 @@ export default class AdminCohortView extends React.Component {
     }
   }
 
+  sanitizeStringForCsv = str => {
+    let res = str;
+    if (str) {
+      // Convert line breaker to dot, multiple spaces to single space
+      res = res.replace(/\n|\r\n|\r+/gm, '.');
+      res = res.replace(/\s+/gm, ' ');
+      res = res.trim();
+    }
+
+    return res;
+  };
```

Comment:
Since both `replace` and `trim` return a new string instead of modifying the original string, you could chain these calls.

```js
sanitizeStringForCsv = str => (str || '')
  .replace(/\n|\r\n|\r+/gm, '.')
  .replace(/\s+/gm, ' ')
  .trim();
```

Reply:
I like this, it is much cleaner!